collect_app/src/main/java/org/odk/collect/android/adapters/SelectMultipleListAdapter.java

```diff
@@ -62,6 +62,8 @@ CheckBox setUpButton(final int index) {
             } else {
                 removeItem(selection);
             }
+
+            widget.widgetValueChanged();
```

Comment:
The same here. Shouldn't it be called in clearAnswer() too?

Reply:
🙏 Fixed and force pushed, thanks!